firestore-counter/POSTINSTALL.md

```diff
@@ -18,17 +18,26 @@ match /databases/{database}/documents/pages/{page} {
 }
 ```
 
-#### Set up a scheduled function
 
-Review the [scheduled function documentation](https://firebase.google.com/docs/functions/schedule-functions) to set up a call to `${function:controller.url}` every minute. You may need to enable some APIs in your Firebase project to use scheduled functions.
+#### Set up a Cloud Scheduler job
 
-As an example, to set up a scheduled function, you can run the following [`gcloud`](https://cloud.google.com/sdk/gcloud/) commands:
+**IMPORTANT:** Note the following about v0.1.1 of this extension:
+- **If you updated your extension from v0.1.0 to v0.1.1:**  We recommend that you edit your Cloud Scheduler job to instead send a message to the extension's Pub/Sub topic, as described in this section. Although it's not recommended, if you leave your Cloud Scheduler job calling `${function:controller.url}`, your extension will continue to run as expected. For more information about the changes for v0.1.1, refer to the [changelog](https://github.com/firebase/extensions/blob/next/firestore-counter/CHANGELOG.md).
+- **If you installed this extension for the first time at v0.1.1:** Follow the instructions as described in this section.
+
+
+Set up a [Cloud Scheduler job](https://firebase.google.com/docs/functions/schedule-functions) to regularly send a message to the extension's Pub/Sub topic (`${param:EXT_INSTANCE_ID}`). This Pub/Sub topic then automatically triggers the Pub/Sub controller function (`${function:controllerPubSub}`). This Pub/Sub controller function is created by the extension and monitors the extension's workload.
```

Comment:
@laurenzlong What is the final name of the Pub/Sub controller function?  It looks like it's "controllerCore"?

Reply:
And I think that we should _probably_ update our copy throughout to just use "controllerCore function" to describe this function, i.e., leave out the "Pub/Sub" bit.  WDYT?

For example:
"... triggers the controllerCore function (...). This controllerCore function is created by..."

Reply:
Yes that sounds good. 

Also `${function:controllerPubSub}` should become `${function:controllerCore}`

Reply:
updated.  thx!